k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala

```diff
@@ -151,6 +151,8 @@ abstract class EndpointsNamer(
               case (oldMap, v1.ServiceAdded(service)) =>
                 val newMap = service.portMappings
                 logEvent.addition(newMap -- oldMap.keys)
```

Comment:
This is mostly unrelated to this change but it just now occurs to me that in large clusters these are potentially expensive set operations.  Might be good to factor this so that the set operations happen inside of a ifTrace block.

Reply:
(here and elsewhere)

Reply:
Yeah, I've wanted to rewrite the logging code for a bit, thanks for reminding me!